api/job/v1beta1/job.proto

```diff
@@ -56,16 +57,20 @@ message ExportJobResponse {
   repeated ExportJobItem jobs = 1;
 }
 
-message GetJobsRequest {
+message GetAvailableJobsRequest {
   // Any non-empty string representing geographic region
   // i.e. san-fran, new-york, alaska, us, global, test, etc.
   // Should be used consistently across all APIs.
-  string area_key = 1;
-  string worker_id = 2;
+  string area_key = 1 [(validate.rules).string = {
+    min_len: 1,
+  }];
+  string worker_id = 2 [(validate.rules).string = {
+    max_len: 36,
```

Comment:
Would be kinda weird to see IDs longer than 36 chars, but maybe better for OME to abide by Postel's Law here.